convene-web/app/views/waiting_rooms/show.html.erb

```diff
@@ -1,6 +1,6 @@
-<div class="bg-blue-100 min-h-screen flex items-center justify-between">
```

Comment:
I'm going to create a ticket for refactoring this view to focus the DOM onto the intent and remove extraneous classe (where possible).

In particular, I want the DOM to look something like:

<div class="waiting-room">
  <div class="room-info">
  
  </div>
  <form>
    <div class="access-code-field --with-errors"> 
      <label>Access Code</label>
      <input type="password" name="waiting_room[access_code]" />
      <p class="input-error">Error Message.</p>
      <p class="input-error">Second Error Message.</p>
    </div>
    <input type="submit" />
  </form>
</div>